src/Loading.svelte

```diff
@@ -7,6 +7,7 @@
   export let fadeIn = false;
   export let margins = false;
   export let condensed = false;
+  export let noWait = false; // To avoid confusion when multiple cursor types are defined
```

Comment:
Why do we want this to be `true` sometimes?

Reply:
I removed the `noWait` prop in [92ef5e7](https://github.com/radicle-dev/radicle-interface/pull/344/commits/92ef5e768fbf64f2ae44e6e98ea592b3811ee901) and hide the `Button` for a `Loading` component in a nice transition